### DISCUSSION TITLE
Omit Author Names and Parentheses in Footnotes

### DISCUSSION THREAD

**Discussion by dstark** [jgm/pandoc#9771] [Q&A] [answered]
Is it (or could it become) possible for some syntax combination _both_ to trigger the omission of an author name _and_ to avoid parentheses around the citation in a footnote style?

As I understand it, author names can be suppressed by adding `-` ahead of the `@` symbol. This works fine for me with "regular" citations. But for citations in footnotes, the suppression doesn't appear to work. Instead, the `-` simply gets added before the citation as in the screenshot below.

<img width="812" alt="Screenshot 2024-05-09 132639" src="https://github.com/Zettlr/Zettlr/assets/205528/656b7bc9-4d4f-4995-acc6-f07b3c0ecfd2">

Or if I wrap the citation in brackets, then the author name suppression works as expected, but parentheses appear.

![image](https://github.com/Zettlr/Zettlr/assets/205528/ee1eaf1e-f75c-4ea0-b16d-08f5fd8e8145)

Any thoughts on this issue would be most helpful! (It does seem related to but perhaps not exactly the same as the question addressed in [this issue](https://github.com/jgm/pandoc/issues/6828).)

**Comment by jgm** [comment, marked as answer]
There isn't a way to do this currently.  I think it's discussed already in issue #7219

**Reply by dstark** [reply]
> I think it's discussed already in issue https://github.com/jgm/pandoc/issues/7219

I agree, @jgm. Sorry I missed that one when searching for the issue. What you describe there as "revert[ing] to the old behavior" does sound like it would address this scenario and at least provide a helpful option for such cases even though, in some styles it might make more sense and in some less.

Thanks so much!

**Reply by jgm** [reply]
I'm going to make the change.

**Reply by dstark** [reply]
@jgm, thank you so much!
